author.php

```diff
@@ -19,54 +19,128 @@
 		<div class="row">
 
 			<!-- Do the left sidebar check -->
-			<?php get_template_part( 'global-templates/left-sidebar-check' ); ?>
+<!--			--><?php //get_template_part( 'global-templates/left-sidebar-check' ); ?>
```

Comment:
Can you delete this, any other commented out code, and the right sidebar check too?